entrypoint.sh

```diff
@@ -9,6 +9,16 @@ then
   chmod 600 /root/.ssh/id_rsa
 fi
 
+if [[ -n "$SSH_KNOWN_HOSTS" ]]
+then
+  mkdir -p /root/.ssh
+  echo "StrictHostKeyChecking yes" >> /etc/ssh/ssh_config
+  echo "$SSH_KNOWN_HOSTS" > /root/.ssh/known_hosts
```

Comment:
```suggestion
  echo "$SSH_KNOWN_HOSTS" > /root/.ssh/known_hosts
  chmod 600 /root/.ssh/known_hosts
```

Let's also set file permissions here.